cpp/src/Ice/SSL/SchannelEngine.cpp

```diff
@@ -1316,15 +1291,19 @@ Ice::SSL::ClientAuthenticationOptions
 Schannel::SSLEngine::createClientAuthenticationOptions(const string& host) const
 {
     return Ice::SSL::ClientAuthenticationOptions{
-        .clientCertificateSelectionCallback =
+        .clientCredentialsSelectionCallback =
             [this](const string&)
         {
-            PCCERT_CONTEXT cert = nullptr;
-            if (_allCerts.size() > 0)
+            for (auto& cert : _allCerts)
             {
-                cert = CertDuplicateCertificateContext(_allCerts[0]);
+                CertDuplicateCertificateContext(cert);
             }
-            return cert;
+
```

Comment:
The defaults for IceSSL configuration are now defined here and don't apply when using the auth options API.